translations/en.json

```diff
@@ -1,878 +1,891 @@
-  "teamProjectAdded": "Project added to team",
-  "teamRemoved": "Team removed from accessing this project",
-  "teams": "Teams",
-  "teamsAndCollaborators": "Teams & Collaborators",
-  "templates": {
-    "apiScanURLFilter": "API Scanner URL Filter",
-    "enterEndpoint": "Enter the API endpoint to scan: eg. subdomain.domain.com. Do not specify the scheme (http://...), port (:443/...) & path (.../users)",
-    "enterAPIEndpoint": "Enter host",
-    "saveFilter": "Save Filter",
-    "addNewUrlFilter": "Add Filter"
-  },
-  "testCredentials": "Test Credentials",
-  "title": "Title",
-  "toDate": "To Date",
-  "tokenCopied": "Token Copied!",
-  "transferCredits": "Transfer Credits",
-  "trialWillBeConverted": "Your free trial will be converted into paid subscription on",
-  "trialWillExpireOn": "Your trial will expire on",
-  "tryAdjustingFilter": "Try adjusting your search or filter",
-  "type": "Type",
-  "unavailable": "unavailable",
-  "unchanged": "Unchanged",
-  "undefined": "undefined",
-  "undo": "Undo",
-  "unknown": "Unknown",
-  "unpaid": "Unpaid",
-  "unreachable": "Unreachable",
-  "untested": "Untested",
-  "update": "Update",
-  "upload": "Upload",
-  "uploadApp": "Upload App",
-  "uploadCount": "Upload Count",
-  "uploadIdpFile": "Upload Idp File",
-  "uploadNewProject": "You can upload a new project by clicking on <b class='bold'>Upload App</b> above and start scanning",
-  "uploadZipFile": "Upload .zip file",
-  "uploaded": "Uploaded",
-  "uploading": "Uploading",
-  "uploads": "Uploads",
-  "uploadStatistics": "Upload Statistics",
-  "urlUpdated": "Successfully updated the url filter",
-  "usd": "USD for",
-  "user": "User",
-  "userActivateTitle": "Activate User",
-  "userActivationChangeMessage": "Are you sure you want to make",
-  "userDeactivateTitle": "Deactivate User",
-  "userRoleUpdated": "User role changed successfully",
-  "userSetting": " user's settings ",
-  "username": "Username",
-  "validUntil": "Valid until",
-  "validity": "Validity",
-  "version": "Version",
-  "versionCode": "version code",
-  "viewAll": "View All",
-  "viewFiles": "View Files",
-  "viewIntegrationInstructions": "View Instructions to Integrate",
-  "viewUploads": "View uploads",
-  "vulnerabilities": "Vulnerabilities",
-  "vulnerability": "Vulnerability",
-  "vulnerabilityDetails": "Vulnerability Details",
-  "vulnerabilityPreferences": "Vulnerability Preferences",
-  "vulnerabilityPreferencesCondition": "Changes made to vulnerability here will be applied to all future analyses of this project",
-  "vulnerabilityPreferencesEdit": "You can edit the vulnerability preferences below",
-  "warning": "WARNING",
-  "week": "Week",
-  "welcome": "Welcome ",
-  "worsened": "Worsened",
-  "xmlMetadata": "XML Metadata",
-  "year": "Year",
-  "yearly": "Yearly",
-  "yes": "Yes",
-  "youCanCreateTeam": "you can create a new team by",
-  "youWillBeChargedOn": "You will be charged on",
-  "yourPlan": "Your Plan"
+	"0sharableCredits": "0 sharable credits",
```

Comment:
Why tabs instead of spaces?

Reply:
Not sure, how it's added there, will be resolved in next push 👍